network.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+function nth_ip() {
+  network=$1
+  idx=$2
+
+  echo `python -c "from ansible.plugins.filter import ipaddr; print(ipaddr.nthhost('"$network"', $idx))"`
```

Comment:
Why the echo?
```suggestion
  python -c "from ansible.plugins.filter import ipaddr; print(ipaddr.nthhost('"$network"', $idx))"
```

Reply:
I guess it doesn’t matter, but I’ve always done it this way as a style thing to make it obvious that’s the value I want to return from a shell function.